README.md

```diff
@@ -70,6 +74,14 @@ let key = "12345678901234567890123456789012".dataUsingEncoding(NSUTF8StringEncod
 MDBabbage.startWithDatastoreAtURL(dir, encryptionKey: key)
 ```
 
+## Self Registering the user
+You can create an account for storing data to AWS S3 buckets.
+
+```swift
+// In CreateAccountViewController.swift
+client.registerSubjectWithEmail(userEmail, password: userPassword, securityQuestionID: userSecurityQuestionID, securityQuestionAnswer: userSecurityAnswer completion:(void (^)(NSError *error))completion
```

Comment:
Can this code be updated to use actual values, similar to how the other code examples are?